pkg/chartrenderer/default.go

```diff
@@ -24,15 +24,15 @@ import (
 	"path/filepath"
 	"strings"
 
+	"helm.sh/helm/v3/pkg/chart"
+	chartloader "helm.sh/helm/v3/pkg/chart/loader"
+	v3chartutil "helm.sh/helm/v3/pkg/chartutil"
+	"helm.sh/helm/v3/pkg/engine"
 	"k8s.io/apimachinery/pkg/version"
 	"k8s.io/client-go/discovery"
 	"k8s.io/client-go/rest"
-	"k8s.io/helm/pkg/chartutil"
-	"k8s.io/helm/pkg/engine"
 	"k8s.io/helm/pkg/ignore"
 	"k8s.io/helm/pkg/manifest"
```

Comment:
Why old helm is used here?

```
         "k8s.io/helm/pkg/ignore"
	"k8s.io/helm/pkg/manifest"
```

Reply:
`k8s.io/helm/pkg/manifest` removed and replaced by helm.sh/helm/v3/pkg/releaseutil pkg.